test_suite.py

```diff
@@ -64,153 +64,222 @@ def run(self, command):
 
 @pytest.fixture
 def docker_runner(docker_client, docker_image):
-    runner = DockerRunner(docker_client, docker_image)
+    runners = []
+    def _docker_runner(language):
+        runner = DockerRunner(docker_client, docker_image(language))
+        runners.append(runner)
+        return runner
+
+    yield _docker_runner
+
+    # I don't love having a list to append to, but don't know how to find the runner
+    # in order to do clean up after the yield
+    if runners: # i.e. if the test called `docker_runner.run(...)`
+        runners[0].container.stop()
+        runners[0].container.remove()
 
-    yield runner
+@pytest.fixture(params=['python'])
+def language(request):
+    param = request.param
 
-    if runner.container: # i.e. if the test called `docker_runner.run(...)`
-        runner.container.stop()
-        runner.container.remove()
+    yield param
```

Comment:
This could be a way to parametrize the test suite, by adding each new language to the list and then each test is run for each given parameter